app/assets/stylesheets/homepage.css.scss

```diff
@@ -16,14 +16,39 @@
     font-weight: 200;
 }
 
-.nav-bar-nk {
+#logo {
+    text-decoration: none;
+    margin-left: -70px ;
+}
+
+.nav-bar-link {
     color: white;
-    font-size: 20px;
+    font-size: 1.1em;
+    text-decoration: none;
+    height: 76px;
+    display: block;
+    padding: 8px 12px 0 12px;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Shorthand form for property `padding` should be written more concisely as `8px 12px 0` instead of `8px 12px 0 12px`